addons/tooltips/script.js

```diff
@@ -510,9 +510,7 @@ export default async function ({ addon, console, msg }) {
     if (addon.settings.get("color-scheme") !== "auto") {
         tippyGlobalOptions.theme = `sa-tooltips-${addon.settings.get("color-scheme")}`
     } else {
-        // I believe that there's a better way to check if dark-www is turned on.
-        // Please tell me if there is.
-        if (document.querySelector("[data-addon-id=dark-www]")) tippyGlobalOptions.theme += "sa-tooltips-dark"
+        if (await (await addon.self.getEnabledAddons()).indexOf('dark-www') + 1) tippyGlobalOptions.theme += "sa-tooltips-dark"

```

Comment:
```suggestion
        if ((await addon.self.getEnabledAddons()).includes("dark-www")) tippyGlobalOptions.theme += "sa-tooltips-dark"
```

Reply:
The +1 is needed because when it does not exist, it returns -1, and -1 is a truthy statement, which I wanted it to be falsy.

The multiple asyncs, yeah that's where it got confused.